MAUI/Calendar/migration.md

```diff
@@ -1105,7 +1105,7 @@ this.Content = calendar;
 <tr>
 <td>{{'[NavigateTo](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.SfCalendar.html#Syncfusion_SfCalendar_XForms_SfCalendar_NavigateTo_System_DateTime_)'| markdownify }}</td>
```

Comment:
DayHeaderFont | Nil
-- | --


[DayHeaderFont](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfCalendar.XForms.MonthViewSettings.html#Syncfusion_SfCalendar_XForms_MonthViewSettings_DayHeaderFont)	Nil

Check forms and update